python/s3-exposed-bucket.py

```diff
@@ -0,0 +1,84 @@
+#
+# This file made available under CC0 1.0 Universal (https://creativecommons.org/publicdomain/zero/1.0/legalcode)
+#
+# Ensure that no buckets are globally accessible.
+# Description: Checks that all buckets restrict access from the global groups.
+#
+# Trigger Type: Change Triggered
+# Scope of Changes: S3:Bucket
+#
+# NOTE: This rule will only find applicable resources once S3 resources are supported by AWS Config.
```

Comment:
Thanks for the enthusiasm in creating a rule for S3 buckets. However, since S3 resources are not currently supported by AWS Config, this could cause user confusion. Could you please remove this until AWS Config supports S3 resources officially?